flink-runtime-web/web-dashboard/src/app/pages/job-manager/log-detail/job-manager-log-detail.component.ts

```diff
@@ -66,6 +75,12 @@ export class JobManagerLogDetailComponent implements OnInit, OnDestroy {
     this.jobManagerService
       .loadLog(this.logName)
       .pipe(
+        catchError(() => {
```

Comment:
What is the reason for this change? What does it have to do with the overall PR?